packages/angular/ssr/test/routes/ng-routes_spec.ts

```diff
@@ -296,6 +276,44 @@ describe('extractRoutesAndCreateRouteTree', () => {
       ]);
     });
 
+    it('should resolve catch all routes for SSG and not add a fallback route if fallback is Server', async () => {
```

Comment:
Question: Why shouldn't this add a fallback route if the fallback is `Server`? Looking at the assertion, it looks to me like it does have a fallback based on `{ route: '/user/*/**', renderMode: RenderMode.Server }`?

Reply:
Typo